server/scsynth/SC_CoreAudio.h

```diff
@@ -268,8 +270,14 @@ class SC_CoreAudioDriver : public SC_AudioDriver {
 
     bool StopStart();
 
+    template <bool isClipping>
```

Comment:
typically we capitalize the first letter of template parameter names, to help distinguish them from ordinary variables